src/osip-monitoring-view/src/test/java/edu/kit/pse/osip/monitoring/view/dialogs/DialogTest.java

```diff
@@ -30,10 +35,32 @@ public void start(Stage arg0) throws Exception {
     
     /**
      * Tests the constructors of the Dialog classes.
+     * 
+     * @throws Exception when something goes wrong.
      */
     @Test
-    public void testDialogs() {
+    public void testDialogs() throws Exception {
         assertNotNull(about);
         assertNotNull(help);
+        final CompletableFuture<Boolean> showed = new CompletableFuture<>();
+        Platform.runLater(() -> {
+            about.show();
+            help.show();
+            showed.complete(true);
+        });
+        Thread.sleep(1000);
+        assertTrue(about.isShowing());
+        assertTrue(help.isShowing());
+        assertTrue(showed.get());
```

Comment:
`get()` blockiert, das heißt du brauchst in der Reihenfolge kein `sleep()`

```
 +        final CompletableFuture<Boolean> showed = new CompletableFuture<>();
 +        Platform.runLater(() -> {
 +            help.show();
 +            showed.complete(true);
 +        });
 +        Thread.sleep(1000);
 +        assertTrue(help.isShowing());
 +        assertTrue(showed.get());
```

vs 

```
 +        final CompletableFuture<Boolean> showed = new CompletableFuture<>();
 +        Platform.runLater(() -> {
 +            help.show();
 +            showed.complete(true);
 +        });
 +        assertTrue(showed.get());
 +        assertTrue(help.isShowing());
```